modules/node/variables.tf

```diff
@@ -13,7 +13,7 @@ variable "ssh_root_public_key" {
 variable "ssh_tech_public_key" {
 }
 
-variable "region" {
+variable "scaleway_region" {
   default = "ams1"
```

Comment:
region is now called `nl-ams`, but you must provide a zone which is  `nl-ams-1`. See the new doc about it  ==> https://www.terraform.io/docs/providers/scaleway/guides/regions_and_zones.html

Reply:
FIxed, thanks!